libcnb-data/src/buildpack.rs

```diff
@@ -226,25 +234,28 @@ libcnb_newtype!(
     /// let stack_id: StackId = stack_id!("heroku-20");
     /// ```
     stack_id,
-    /// buildpack.toml Stack Id. This is a newtype wrapper around a String.
-    /// It MUST only contain numbers, letters, and the characters ., /, and -.
-    /// or be `*`.
+    /// The ID of a stack.
+    ///
+    /// It MUST only contain numbers, letters, and the characters `.`, `/`, and `-`.
     ///
-    /// Use [`std::str::FromStr`] to create a new instance of this struct.
+    /// Use the [`stack_id`](crate::buildpack_id) macro to construct a `StackId` from a
+    /// literal string. To parse a dynamic string into a `StackId`, use
+    /// [`str::parse`](str::parse).
     ///
     /// # Examples
     /// ```
-    /// use std::str::FromStr;
-    /// use libcnb_data::buildpack::StackId;
+    /// use libcnb_data::buildpack::BuildpackId;
```

Comment:
This and the below references buildpack ID rather than stack ID

Reply:
I'm sorry - I don't know how this slipped by. 🙄